File: backend/backend/model/Building.py
from django.db import models
from User import User
from Manual import Manual
from LocatieEnum import LocatieEnum


class Building(models.Model):
    adres = models.TextField()
    syndicus = models.ForeignKey(
        User,
        on_delete=models.DO_NOTHING
    ),
    owners = ArrayField(User),
    ivago_klantnr = models.IntegerField()
    manual = models.ForeignKey(
        Manual,
        on_delete=models.CASCADE
    )
    # TODO Add TrashContainer model
    # containers = ArrayField(TrashContainers)
    locatie = models.ForeignKey(
        LocatieEnum,
        on_delete=models.DO_NOTHING
    )

File: frontend/src/api/models/Building.ts
import BuildingManual from "@/api/models/BuildingManual";

// TODO fix, object not correct with what backend sends
export default class Building {
  name: string;
  id: number;
  adres: string;
  ivago_klantnr: number;
  buildingID: string;
  manual: BuildingManual;
  containers: null; // TODO Add model
  location: null; // TODO Add model
}
